stemcell_builder/stages/bosh_monit/assets/01-restrict-monit-api-access

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# THESE SCRIPTS ARE EXECUTED IN LEXICAL ORDER. MAKE SURE TO CHECK IF ANYTHING ELSE (CURRENTLY BOSH FOR NATS) WRITES OTHER CGROUP BASED RULES TO AVOID INTERFERENCE. IT'S PROBABLY A GOOD IDEA TO -A DROPS AND -I ACCEPTS
```

Comment:
I notice that you followed this recommendation in the script that you wrote: `02-restrict...`, but didn't modify this script to match the recommendation.

If it's a good recommendation, I'd suggest updating this script to match it, but I'm having a hard time understanding why you'd prepend `ACCEPT` rules and append `DROP` rules, rather than appending both.

I'm probably having a bad brain day... would you be so kind as to walk me through the reasoning on Monday?

NOTE: If you resolve my other comment, I'm fine with this PR, so don't block the merge because of _this_ comment... but I do -at some point- want to understand the reasoning for the difference in handling between `DROP` and `ACCEPT` rules.